src/coreclr/tests/dir.sdkbuild.props

```diff
@@ -9,7 +9,7 @@
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <EnableDefaultItems>false</EnableDefaultItems>
-    <Platform>$(BuildArch)</Platform>
+    <Platform>AnyCPU</Platform>
```

Comment:
Was this change required because `BuildArch` has different value in this case?

Reply:
`BuildArch` is equal to actual arch specified for build (i.e. `arm`, `x64`)

Reply:
Can this line be deleted?

Reply:
The same error with incorrect format appears if this line is removed